install.sh

```diff
@@ -1,5 +1,11 @@
 #!/bin/bash
 
+sudo apt-get clean
+sudo mv /var/lib/apt/lists /tmp
```

Comment:
I hadn't noticed this before. I'm not sure why we need to keep this in /tmp. Can we just get rid of it?